bin/getTiming.sh

```diff
@@ -160,20 +159,20 @@ fi
 workDir="${PRM_ROOT_DIR}"
 projectsDir="${workDir}/projects/"
 logsDir="${workDir}/logs/"
-logsDirSourceServer="/groups/${GROUP}/${SCR_LFS}/logs/"
+logsDirSourceServer="/groups/${group}/${SCR_LFS}/logs/"
 
 cd "${projectsDir}"
-find * -nowarn -mtime -40 -type d -maxdepth 0 -exec ls -d {} \; > "${logsDir}/AllProjects40days.txt"
+find ./* -nowarn -mtime -40 -type d -maxdepth 0 -exec ls -d {} \; > "${logsDir}/AllProjects40days.txt"
 SAMPLESHEET_SEP=","
-echo -e "unique_id,rawdataname,project,total_min,total_hours,machine,numberofSamples,startTime,finishedTime,copyRawDataToPrmDuration,pipelineDuration,copyProjectDataToPrmTiming,demultiplexingDuration" > ${logsDir}/status_timing.csv
-while read project
+echo -e "unique_id,rawdataname,project,total_min,total_hours,machine,numberofSamples,startTime,finishedTime,copyRawDataToPrmDuration,pipelineDuration,copyProjectDataToPrmTiming,demultiplexingDuration" > "${logsDir}/status_timing.csv"
+while read -r project
 do
 	echo "start"
 	declare -a sampleSheetColumnNames=()
 	declare -A sampleSheetColumnOffsets=()
-	numberOfSamples=$(ls ${projectsDir}/${project}/run01/results/variants/*.final.vcf.gz | wc -l)
+	numberOfSamples=$(find "${projectsDir}/${project}/run01/results/variants/" -name "*.final.vcf.gz" | wc -l)
```

Comment:
Een ```*``` tussen dubbele quotes kan ge-expandeerd worden door de (Bash) shell.
Een ```*``` tussen enkele quotes wordt als een letterlijke ```*``` gezien en niet ge-expandeerd door de shell.
Is in feite hetzelfde verschil als voor variabelen:
```"${var}"``` wordt ge-expandeerd naar de waarde van die variable, maar ```'${var}'``` is de letterlijke tekst ${var}.
Bij een ```ls .... *.final.vcf.gz``` moet de ```*``` ge-expandeerd worden door de shell. Dat gaat zonder quotes of met veiliger met dubbele quotes, maar niet met enkele. Bij de find is de ```*``` onderdeel van een regex pattern en om te voorkomen dat de shell er nog mee aan de haal gaat kan dat veiliger als:
```'*.final.vcf.gz'```